Orange/tests/test_variable.py

```diff
@@ -247,6 +247,11 @@ def test_colors(self):
         self.assertEqual(len(var.colors), 4)
         np.testing.assert_array_equal(var.colors[2], user_defined)
 
+    def test_no_nonstringvalues(self):
+        self.assertRaises(TypeError, DiscreteVariable, "foo", values=["a", 42])
+        a = DiscreteVariable("foo", values=["a", "b", "c"])
+        self.assertRaises(TypeError, DiscreteVariable.add_value, 42)
```

Comment:
This was probably intended to be `self.assertRaises(TypeError, a.add_value, 42)`